### DISCUSSION TITLE
footer添加多行自定义文本

### DISCUSSION THREAD

**Discussion by wzwzx** [MaLuns/hexo-theme-async#49] [Q&A 疑难杂症] [answered]
![Screenshot_20230102-002550_Via](https://user-images.githubusercontent.com/69845256/210177839-f630cea4-636c-4276-957b-bfa7477bc515.png)
添加多行文本后<br>换行都换不下去，电脑端显示挺正常，手机端会导致显示彻底混乱

**Comment by MaLuns** [comment, marked as answer]
收到反馈，会在下个版本，对这里样式优化下

**Comment by MaLuns** [comment]
已优化，v1.2.12

![image](https://user-images.githubusercontent.com/31614024/210326987-6e76f91a-495e-4cf4-a162-67f262fec595.png)

**Reply by wzwzx** [reply]
一看到更新我就整上力，比以前好看很多啦
![Screenshot_20230103-235137_Via](https://user-images.githubusercontent.com/69845256/210392702-cb38851e-cad6-432b-a804-bb4897984703.png)
